Extras/request-logger/request-logger.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE request_log (
```

Comment:
We should follow the directory structure and name convention of either `odin` or `tormor`. So we can use those to handle the migrate.

Like `/Schema/request-logger/001-initial.blue.sql`